Makefile

```diff
@@ -117,11 +120,6 @@ _go_webdriver_test: var-BROWSER java go_build_test xvfb geckodriver webserver_de
 web_components_test: xvfb firefox chrome webapp_node_modules_all psmisc
 	util/wct.sh $(USE_FRAME_BUFFER)
 
-# Dependencies for running dev_appserver.py.
-webserver_deps: webapp_deps dev_appserver_deps
-
-webapp_deps: go_build webapp_node_modules
```

Comment:
This is also used by `deploy.sh`

Reply:
Good catch. That make rule is out of date anyway.